package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pino-noir",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
this should be a minor increase not a patch, since it's an API extension

Reply:
can you give me a hint here: how do I run the benchmark? I looked for a command but didn't found it anywhere :(
I would include this as a script tag afterwards.

Reply:
Regarding the tests: there is one called "masks according to supplied censor function" which covers the new functionality and and another one "masks according to supplied censor" covering the old functionality.
The coverage report displays only line 11 as uncovered:
index.js |      100 |    98.04 |      100 |      100 |             11 |

Reply:
@yellowbrickc you can run the benchmarks with `node benchmarks`. You can also add that to the package.json if you like.